lib/ops/opsservice/install.go

```diff
@@ -886,6 +886,10 @@ func (s *site) newProvisioningToken(operation ops.SiteOperation) (token string,
 		OperationID: operation.ID,
 		UserEmail:   agentUser.GetName(),
 	}
+	if operation.Type == ops.OperationExpand {
+		// Set a TTL for expand provisioning token.
+		tokenRequest.Expires = s.clock().UtcNow().Add(24 * time.Hour)
```

Comment:
I'm curious as well on why 24 hours? I'm not all that familiar with this code, so without tracing it all through, what happens after the 24 hours?

Reply:
Nothing, really. Before we removed this the ttl was 1 hour which was a bit short b/c if expand operation was for some reason created but not started/completed within 1 hour, the token would become invalid. 24 hours just gives more breathing room.